testdata/p4_16_samples_outputs/bvec-hdr-bmv2.p4.entries.txt

```diff
@@ -0,0 +1,46 @@
+updates {
+  type: INSERT
+  entity {
+    table_entry {
+      table_id: 33562569
+      match {
+        field_id: 1
+        exact {
+          value: "\001"
+        }
+      }
+      action {
+        action {
+          action_id: 16837978
+          params {
+            param_id: 1
+            value: "\000\001"
```

Comment:
This is a question for my curiosity, not necessarily a problem with the commit.  Should these values use the minimum-length encoding specified in the P4Runtime spec for unsigned integers?  They do not seem to be, but the value looks correct.